datasource/curricle.go

```diff
@@ -81,6 +81,8 @@ func (s *SearchCurricle) Fetch(page uint) (courses []Course, err error) {
 			Description:        sanitizeHtml(node["courseDescriptionLong"].(string)),
 			Instructors:        instructors,
 			MeetingPatterns:    meetingPatterns,
+			GenEdArea:          getGenEdInfo(node["courseAttributes"].([]interface{})),
+			DivisionalDist:     getDivisionalInfo(node["courseAttributes"].([]interface{})),
```

Comment:
Can you use `any` for consistency?
```suggestion
			GenEdArea:          getGenEdInfo(node["courseAttributes"].([]any)),
			DivisionalDist:     getDivisionalInfo(node["courseAttributes"].([]any)),
```